### PR TITLE
Add VTEX eslint configs

### PR DESCRIPTION
**Pull request by gabrielvincent**
This pull request removes deprecated eslint rules and adds `eslint-config-vtex` instead

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
     "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "babel-preset-es2015": "^6.24.1",
     "babel-preset-stage-2": "^6.24.1",
-    "eslint": "^4.12.1"
+    "eslint": "^4.12.1",
```
Comment (kaisermann) [inline package.json:20]: Update this to the last version (6)

**Comment by kaisermann** [comment]
@gabrielvincent Still need the `.prettierrc` file 😁

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+## [Unreleased]
+### Added
+- CHANGELOG.md
+- Prettier
+- VTEX eslint rules
+### Changed
+- Remove deprecated eslint rules
+- Update eslint version
+- Remove semicolons
```
Comment (kaisermann) [inline CHANGELOG.md:16]: Are you going to release a new version? These chores are not necessarily relevant to the changelog 🤔

Comment (gabrielvincent) [reply CHANGELOG.md:16]: I will  release a new version and since that this had no changelog, I figured I should add one.

**Pull request merged**